CMakeLists.txt

```diff
@@ -2310,30 +2312,23 @@ endif()
 # Live Broadcasting (Shoutcast)
 option(BROADCAST "Live Broadcasting (Shoutcast) support" ON)
 if(BROADCAST)
-  find_package(Shout)
-  if(UNIX AND NOT APPLE)
-    # Check if system lib is at least 2.4.6 and not suffering bugs
-    # https://bugs.launchpad.net/mixxx/+bug/1833225
-    # https://bugs.launchpad.net/mixxx/+bug/1913579
-    if(Shout_FOUND AND Shout_VERSION VERSION_LESS 2.4.4)
-        message(STATUS "Installed libshout version: ${Shout_VERSION} is suffering from bugs lp1833225 and lp1913579")
-    elseif(Shout_FOUND AND Shout_VERSION VERSION_LESS 2.4.6)
-        message(STATUS "Installed libshout version: ${Shout_VERSION} is suffering from bug lp1913579")
-    endif()
-    if(NOT Shout_FOUND OR Shout_VERSION VERSION_LESS 2.4.6)
-      # Fall back to internal libraray in the lib tree
-      message(STATUS "Using internal libshout")
-      add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/lib/libshout")
-      target_include_directories(mixxx-lib SYSTEM PUBLIC lib/libshout/include)
-      target_link_libraries(mixxx-lib PUBLIC shout_mixxx)
-    else()
-      target_link_libraries(mixxx-lib PUBLIC Shout::Shout)
-    endif()
+  find_package(Shoutidjc)
+  # Check if system lib is at least 2.4.6 and not suffering bugs
+  # https://bugs.launchpad.net/mixxx/+bug/1833225
+  # https://bugs.launchpad.net/mixxx/+bug/1913579
+  if(Shoutidjc_FOUND AND Shoutidjc_VERSION VERSION_LESS 2.4.4)
+      message(STATUS "Installed libshout-idjc version: ${Shoutidjc_VERSION} is suffering from bug lp1833225")
+  elseif(Shoutidjc_FOUND AND Shoutidjc_VERSION VERSION_LESS 2.4.6)
+      message(STATUS "Installed libshout version: ${Shout_VERSION} is suffering from bug lp1913579")
```

Comment:
This assumes upstream will fix the bug. I have low confidence this will happen considering it has been broken for 5 years.

Reply:
Ping. Let's just use the internal library unless the system has 2.4.1

Reply:
After our experience with upstream I agree. Don't expect anything from them.

Reply:
I don' really expect anything either. However I would like to keep our library as fall back, because this prevents Linux package maintainer from ripping of the internal library.